### DISCUSSION TITLE
More then one interpreter?

### DISCUSSION THREAD

**Discussion by TheLocalPub** [Vdauphin/HeartsAndMinds#1601] [Question and answer] [answered]
As per the title.
I'm looking to create several interpreters. 
Is it possible?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Yes, just copy and paste the current interpreter slot 
(you just need to set the variable visible in the init box: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/mission.sqm#L786)

Cheers

**Comment by TheLocalPub** [comment]
Hmm, okay.
So to clarify, I can have several units with the same variable as "interpreter", if I set it via the units init?

**Reply by Vdauphin** [reply]
Yes

**Reply by TheLocalPub** [reply]
Thank you.

I did make another dicussion thread before this one about updating the github with better documentation on how to do stuff.
This is another thing that could be added for example. There was nothing in the current documentation to help with setting up the interpreter.

**Reply by Vdauphin** [reply]
Fill free to update it!

**Reply by TheLocalPub** [reply]
Have you considered actually using the github wiki feature?
Like my community here https://github.com/Global-Conflicts-ArmA/Olsen-Framework-Arma-3/wiki I maintain this wiki for our mission framework.

I've currently done quite a few edit to the H&M mission, having now completely converted into to an Arma 2 operation arrow head theme style COIN mission. 
I'm also potentially going to look at mounting our communities framework onto this mission and getting extra features and potential.

**Reply by Vdauphin** [reply]
> Have you considered actually using the github wiki feature?

I used it before but discard because you can't update the code and the doc at the same time which lead to inconsistency between release

Please create a new discussion if you want to proceed further
